tests/unit/test_cell_group.cu

```diff
@@ -0,0 +1,36 @@
+#include "../gtest.h"
+
+#include <cell_group.hpp>
+#include <common_types.hpp>
+#include <fvm_multicell.hpp>
+#include <util/rangeutil.hpp>
+
+#include "../test_common_cells.hpp"
+
+using fvm_cell =
+    nest::mc::fvm::fvm_multicell<nest::mc::gpu::backend>;
+
+nest::mc::cell make_cell() {
+    using namespace nest::mc;
```

Comment:
`using namespace` here, but `nest::mc::cell` below, and the whole file is not itself an include file...
Can we just have a `using namespace nest::mc;` after the '#include' directives and use a different variable name than `cell` inside `make_cell()`? There is an overabundance of colons.